data-solutions/data-platform-foundations/01-environment/main.tf

```diff
@@ -31,8 +31,9 @@ module "project-datamart" {
     "storage.googleapis.com",
     "storage-component.googleapis.com",
   ]
-  iam = {
-    "roles/editor" = [module.sa-services-main.iam_email]
+
+  iam_additive = {
```

Comment:
did you try running `apply` with an empty state? `iam_additive` is not safe with dynamic values.

Reply:
True, but `iam_email` is computed [statically](https://github.com/terraform-google-modules/cloud-foundation-fabric/blob/master/modules/iam-service-account/outputs.tf#L32) now. This particular usage should be fine

Reply:
The main issue with authoritative binding for roles/editor was that it removed the binding for some Google owned SA (i.e. Google APIs Service Agent). This leaded to issues when deploying Composer, for instance. terraform apply works OK in a from the scratch deployment